XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/DefaultMixedRealityControllerDataProvidersProfile.asset

```diff
@@ -69,30 +70,33 @@ MonoBehaviour:
     name: Oculus Controller Data Provider
     priority: 8
     runtimePlatform: 67
-    configurationProfile: {fileID: 11400000, guid: 440e002106df2af40806c7b351290415, type: 2}
+    configurationProfile: {fileID: 0}
   - instancedType:
       reference: XRTK.Providers.Controllers.Simulation.SimulatedControllerDataProvider,
         XRTK
     name: Simulated Controller Data Provider
-    priority: 1
+    priority: 9
     runtimePlatform: 1
-    configurationProfile: {fileID: 11400000, guid: 6222a8cbe708b06438176694991e3e56, type: 2}
+    configurationProfile: {fileID: 11400000, guid: 6222a8cbe708b06438176694991e3e56,
+      type: 2}
   - instancedType:
       reference: XRTK.WindowsMixedReality.Controllers.Hands.WindowsMixedRealityHandControllerDataProvider,
         XRTK.WindowsMixedReality
     name: Windows Mixed Reality Hand Controller Data Provider
-    priority: 1
+    priority: 10
     runtimePlatform: 16
-    configurationProfile: {fileID: 11400000, guid: 8a5d0fbb07851a2409d0177ff1f998c7, type: 2}
+    configurationProfile: {fileID: 0}
```

Comment:
Some loss in parent profile configuration?

Reply:
No it was the wrong profile assigned